install.sh

```diff
@@ -1,34 +1,23 @@
 #!/usr/bin/env sh
 
 systemctl -q is-active log2ram  && { echo "ERROR: log2ram service is still running. Please run \"sudo service log2ram stop\" to stop it."; exit 1; }
+[ "$(id -u)" -eq 0 ] || { echo "You need to be ROOT (sudo can be used)"; exit 1; }
 
+# log2ram
+install -m 644 log2ram.service /etc/systemd/system/log2ram.service
+install -m 755 log2ram /usr/local/bin/log2ram
+install -m 644 log2ram.conf /etc/log2ram.conf
+install -m 644 uninstall.sh /usr/local/bin/uninstall-log2ram.sh
+systemctl enable log2ram
 
-if [ "$(id -u)" -eq 0 ]
-then
-  [ -d /usr/local/bin/ ] || mkdir /usr/local/bin/
```

Comment:
Don' remove this line, I need that on recent Ubuntu where the folder isn't created by default

Reply:
ups, missed that. Fixed